packages/flutter/lib/src/foundation/persistent_hash_map.dart

```diff
@@ -0,0 +1,388 @@
+// Copyright 2014 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+/// A collection of key/value pairs, from which you can retrieve a value
```

Comment:
nit: avoid "you", see https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo#use-the-passive-voice-recommend-do-not-require

Reply:
This sentence is copied from `Map` dartdoc. I am open to suggestions on how to rewrite it better.